src/Teams/beta/custom/GetMgTeamsAppPreApproval_Get.cs

```diff
@@ -0,0 +1,409 @@
+namespace Microsoft.Graph.Beta.PowerShell.Cmdlets
+{
+    using Microsoft.Graph.Beta.PowerShell.TeamsInternal;
+    using System;
+    using System.Linq;
+    using static Microsoft.Graph.Beta.PowerShell.Runtime.Extensions;
+
+    /// <summary>
+    /// Get Preapproval policy associated with a Teams App.
+    /// </summary>
+    [global::System.Management.Automation.Cmdlet(global::System.Management.Automation.VerbsCommon.Get, @"MgBetaTeamsAppPreApproval_Get")]
+    [global::System.Management.Automation.OutputType(typeof(Microsoft.Graph.Beta.PowerShell.Models.IMicrosoftGraphTeamsAppPreApproval))]
+    [global::Microsoft.Graph.Beta.PowerShell.Description(@"Get Preapproval policy associated with a Teams App.")]
+    [global::Microsoft.Graph.Beta.PowerShell.Generated]
+    public partial class GetMgTeamsAppPreApproval_Get : global::System.Management.Automation.PSCmdlet,
```

Comment:
Let's change the file and class name to `GetMgBetaTeamsAppPreApproval` to match the name convention of beta commands.

Reply:
Renamed